apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -19,9 +19,40 @@ FluentUI supports browsers that are at or above the indicated versions below:
 | >=14.5        | >=84               | >=16    |
 
 FluentUI will ensure that no browser, EcmaScript(ES), or CSS features that are incompatible with the support matrix laid out above are introduced.
-FluentUI will not be providing polyfills for features that we expect our browser support matrix to already support. This means that consumers who require the
-use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
-are expected to implement their own polyfills or use a third-party one.
+FluentUI will not be providing our own polyfills for features that we expect our browser support matrix to already support.
+There are often well-tested third-party polyfills available that applications can use to fill in the gaps.
+Consumers who require the use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. ResizeObserver),
+will need to implement or use a third-party polyfill to provide the functionality. See the partial support section below for features
+that may need to be polyfilled.
+
+## Partial Support:
+
+FluentUI provides partial support for the browsers below which will have limitations that may require consumers to use/implement polyfills to cover the support gaps.
+
+**Desktop Browsers:**
+
+| Edge | Firefox | Chrome | Safari | Opera | Internet Explorer |
+| ---- | ------- | ------ | ------ | ----- | ----------------- |
+| >=79 | >=69    | >=79   | >=13.1 | >=64  | Not Supported     |
+
+**Mobile Browsers:**
+
+| Safari on iOS | Chrome for Android | Samsung |
+| ------------- | ------------------ | ------- |
+| >=13.4        | >=79               | >=14    |
+
+List of features that are currently used in FluentUI that aren't supported out of the box by the partial support matrix above:
+
+**Unsupported ECMAScript Features:**
+
+1. Nullish Coalescing: Unsupported by Chrome, Edge, Firefox, Opera
```

Comment:
**thought:** AFAIK this could not be polyfilled as its a transpile time transforms - meaning partners will need to manually down-transpile our packages within their tool chain. Is that intended ? if yes can we add it for better context to this document ?

Reply:
Yeah it's intended that they'll need to transpile down our packages themselves if needed, i'll update the document to make this clearer.